requirements.txt

```diff
@@ -1,4 +1,4 @@
-torch>=1.6
+torch>=1.7.1
```

Comment:
Currently GPU tests are failing with:
```E   OSError: /usr/local/lib/python3.7/dist-packages/torchtext/_torchtext.so: undefined symbol: _ZNK3c104Type14isSubtypeOfExtESt10shared_ptrIS0_EPSo```
I assume it has something to do with the update to the torch version

Reply:
I'm unsure, but we have to bump pytorch and torchvision to 1.7.1 and 0.8.2 to use RetinaNet